### ISSUE TITLE
Train on load?

### ISSUE THREAD

**Issue by mzed** [mzed/rapid#6]
When an object is instantiated with an existing dictionary, should it automatically train? What order do these things happen when a patch is loaded?

**Comment by francesco-di-maggio** [comment]
This is an interesting feature. It would be nice indeed to have rapid train automatically a model when instantiated with an existent dictionary file name. Alternatively, in Max one could dynamically read a dict file (or use a "loadmess #1" > prepend read), inform the rapid object prepending by the "trainingDict" message, and then train rapid. Like so. Food for thoughts.

<img width="791" alt="3" src="https://user-images.githubusercontent.com/64175407/208988740-8acf514e-daf1-457a-b371-c9997ba5dba4.png">

<img width="1512" alt="2" src="https://user-images.githubusercontent.com/64175407/208988719-4cb28719-ac8c-4797-8d32-ffccc98f8802.png">

**Comment by mzed** [comment]
An object with an @trainingDict attribute will train on that with just a train message. So, "train" could be triggered by loadbang/loadmess.  

I'm not 100%  sure of the ordering (when is that dictionary ready?), so there would need to be some testing. Also, a really huge dictionary could cause a substantial training time.  These are my concerns.  If read/write models is working properly, then maybe that's a better solution.

**Comment by francesco-di-maggio** [comment]
Good point. Maybe the <defer> and <deferlow> objects could come in hand in this case? So far I used this workflow of dict reading > rapid addressing > train message deferring, and never had issues. But certainly something to test further. On another prospective, objects like buffer~ seem to implement what - I think - reflects the intentions behind this issue: i.e. automatically load target content when instantiated with an existent buffer name...?
